app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+  def create
+    @user = Post.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new(comment_params)
+    @comment.author = current_user
+    @comment.post = @post
+
+    if @comment.save
```

Comment:
- [ ] Your `CommentsController` should have `new` and `create` actions. You currently haven't created the `new` action 👍

Reply:
Dear @codepantha ,
I am so grateful for the effort you have taken to view my project.
The reason to why i didn't have the new action is simply because, the `new ` action Controller will redirect me to `http://127.0.0.1:3000/users/1/posts/5/comments/new` to create add a new `comment`.
Considering the above , it would be of good practice to have `add comment form` on the same `root` of the post.
If there are solid reasons to why i should change this to your preferences, you can let me know to change that. Thank you.

Reply:
@AlphaNtihinduka, It is not my preference, but rather a core requirement for this project milestone. Sometimes, there may be a better way to do things but the whole point is to be able to execute specifications to the letter, and in this case, the comments have to be created on its own new page 👍